packages/contracts/contracts/voting/allowlist/AllowlistVoting.sol

```diff
@@ -0,0 +1,235 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/utils/Checkpoints.sol";
+
+import "../../utils/UncheckedMath.sol";
+import "../majority/MajorityVotingBase.sol";
+
+/// @title AllowlistVoting
+/// @author Aragon Association - 2021-2022.
+/// @notice The majority voting implementation using an list of allowed addresses.
+/// @dev This contract inherits from `MajorityVotingBase` and implements the `IMajorityVoting` interface.
+contract AllowlistVoting is MajorityVotingBase {
```

Comment:
Why not using `MajorityVoting` instead of `MajorityVotingBase`?

Reply:
Explained here: https://github.com/aragon/core/pull/62#discussion_r929652902
Closing this conversation as a duplicate.